libsolidity/formal/ModelCheckerSettings.h

```diff
@@ -153,7 +153,7 @@ struct ModelCheckerSettings
 	ModelCheckerEngine engine = ModelCheckerEngine::None();
 	ModelCheckerInvariants invariants = ModelCheckerInvariants::Default();
 	bool showUnproved = false;
-	smtutil::SMTSolverChoice solvers = smtutil::SMTSolverChoice::All();
+	smtutil::SMTSolverChoice solvers = smtutil::SMTSolverChoice::Z3();
```

Comment:
Changed the default to `z3` because it's the only solver that both BMC and CHC can use plus it's the one we ship best.

Reply:
Yea, will add.